src/tsickle.ts

```diff
@@ -630,6 +638,21 @@ class Annotator extends ClosureRewriter {
     return toArray(reexports.keys());
   }
 
+  private writeModuleSpecifier(moduleSpecifier: ts.Expression) {
+    let moduleId = (moduleSpecifier as any).text;
```

Comment:
I think you probably need to do some sort of switch on moduleSpecifier.kind here, to make sure we produce a good error message in the case where it's something we don't expect.  See other calls to `this.errorUnimplementedKind`.